src/convertToObject.js

```diff
@@ -6,7 +6,36 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  if (sourceString.length === 0) {
+    return {};
+  }
+
+  const newStyles = sourceString.trim();
+  const stylesArray = newStyles.split(';');
+  const arr = [];
+
+  for (let i = 0; i < stylesArray.length; i++) {
+    if (stylesArray[i].trim().length !== 0) {
+      arr.push(stylesArray[i].replace('\n', ''));
+    }
+  }
+
+  const resultArray = [];
```

Comment:
[CODE STYLE] - The variable `resultArray` is also not descriptively named. A better name would be `stylePairs` or something similar that indicates that this array holds key-value pairs of styles.